src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +38,34 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo.id}`}
             </div>
+            {selectedTodo && (
+              /* eslint-disable-next-line jsx-a11y/control-has-associated-label */
+              <button
+                type="button"
+                className="delete"
+                data-cy="modal-close"
+                onClick={() => setSelectedTodo(null)}
+              />
+            )}
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button
-              type="button"
-              className="delete"
-              data-cy="modal-close"
-            />
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {`${selectedTodo.title}`}
             </p>
 
             <p className="block" data-cy="modal-user">
-              {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
+              <strong className="has-text-danger">
+                {statusMessage}
+              </strong>
```

Comment:
`{statusMessage}` should change its color (by changing the class of the tag) depending on which status the todo has (class `has-text-danger` for an active todo and `has-text-success` for a completed one) ![KEKWait (2)](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/146858277/5d0e0fe7-e65f-4118-8313-afbb249509c3)

Reply:
> `{statusMessage}` should change its color (by changing the class of the tag) depending on which status the todo has (class `has-text-danger` for an active todo and `has-text-success` for a completed one) ![KEKWait (2)](https://private-user-images.githubusercontent.com/146858277/290252285-5d0e0fe7-e65f-4118-8313-afbb249509c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0ODQ5ODcsIm5iZiI6MTcwMjQ4NDY4NywicGF0aCI6Ii8xNDY4NTgyNzcvMjkwMjUyMjg1LTVkMGUwZmU3LWU2NWYtNDExOC04MzEzLWFmYmIyNDk1MDljMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxM1QxNjI0NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZjFmY2I0NjA1ZGJmZjM0YzEyMDliZTM2MTAzOGU4ZDMwM2IxNzY5OGZlMGQ2NGI0MDgyOTNkYzhmZjJlMmI2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.MrrNJH7kIiZ2Ujk-UW1tLvQCBAxU9VkGoALLte66MQE)

⬆️